WORKING_MODE.md

```diff
@@ -27,7 +27,7 @@ Resigning members may nominate a candidate to replace them, subject to the AC's
 
 ## Representation
 
-Members are representing themselves, not their employer.
+Members are representing themselves, not their employer. For more information about AC Member Expectations, please see [Member Expectations](https://github.com/ampproject/meta-ac/blob/master/MEMBER_EXPECTATIONS.md).
```

Comment:
Let's avoid casing which leans towards legalese (and transactional relationships).

```suggestion
Members are representing themselves, not their employer. For more information about AC member expectations, please see [Member Expectations](https://github.com/ampproject/meta-ac/blob/master/MEMBER_EXPECTATIONS.md).
```